bindings/pyroot_experimental/PyROOT/python/ROOT/_facade.py

```diff
@@ -50,7 +50,13 @@ def _fallback_getattr(self, name):
         try:
             return getattr(gbl_namespace, name)
         except AttributeError:
-            return getattr(self.__dict__['ROOT_ns'], name)
+            try:
+                return getattr(self.__dict__['ROOT_ns'], name)
+            except AttributeError:
+                if not gROOT.FindObject(name):
```

Comment:
Watch out, if the lookup succeeds you are doing it twice (once in line 56 and again in line 59).
Why not just `res = gROOT.FindObject(name)` and then you raise the error or you return the object depending on the result?
Also, could you please update the comment here:
https://github.com/root-project/root/blob/ee64c1df70155411e951b189b288ff14d3ea6092/bindings/pyroot_experimental/PyROOT/python/ROOT/_facade.py#L48
explaining this third lookup that you are doing (what it is, examples of where it looks).